packages/regl-worldview/src/stories/assertionStories/Worldview.stories.js

```diff
@@ -234,4 +238,80 @@ stories
         expect(result[0].object).toEqual(cube);
       },
     })
+  )
+  .add(
+    `Firing two mouse events at the same time does not cause an error`,
+    assertionTest({
+      story: (setTestData) => (
+        <WorldviewWrapper onMouseDown={(_, clickInfo) => setTestData(clickInfo)}>
+          <Cubes>{[cube]}</Cubes>
+        </WorldviewWrapper>
+      ),
+      assertions: async (getTestData) => {
+        emitMouseEvent("mousedown");
+        await emitMouseEvent("mousedown");
+        const result1 = await getTestData();
+        expect(result1.objects[0].object).toEqual(cube);
+      },
+    })
+  )
+  .add(
+    `(cached hitmap test) A second event at the same point correctly selects the same object`,
```

Comment:
Is there some way to assert that this actually uses the cache, instead of only testing that you get the right data? Not super big deal, but if there's an easy way then let's add that.

Reply:
I looked into it and unfortunately it isn't easy to do.